features/seeing_refund_payments_on_admin_order_view.feature

```diff
@@ -26,3 +26,10 @@ Feature: Seeing refund payments on admin order view
     Scenario: Seeing refund payment on order view
         When I view the summary of the order "#00000022"
         Then I should see 1 refund payment with status "New"
+
+    @ui
+    Scenario: Having all credit memos listed on the order page in ascending order
```

Comment:
Still, I would be for extracting this scenario to a separate file, because credit memo is a different entity than refund payment and doesn't fit this feature file

Reply:
There's been a misunderstanding, I thought you said to put it in this file, I'm going to create a new one then